test/integration/metrics/metrics_test.go

```diff
@@ -119,7 +116,7 @@ func TestAPIServerMetrics(t *testing.T) {
 	}
 
 	// Make a request to a deprecated API to ensure there's at least one data point
```

Comment:
... this seems rather fragile, but I guess we can't easily inject a fake deprecated API ...

Reply:
yeah I'll create an issue for this. Don't want to address it in this PR but always being forced to update it every couple of releases is not ideal.

Reply:
Actually discussed briefly with @jpbetz that since we run api-server in-process we could probably ... actually register an integration-test-only deprecated API that will be perpetually deprecated?